improve the help documents (#132)
kubernetes-sigs/kwok: docs/examples/kwok-manage-nodes-and-pods.md

EOF
```

> After the node is created, the node is in the `Ready` state:

Reviewer: ```suggestion
After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state.
```

> After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state.

Reviewer: ```suggestion
After the Node is created, Kwok will maintain its heartbeat and keep it in the `ready` state:
```

EOF
```

> After the creation, we can see that all the Pods are in the `Runing` state:

Reviewer: ```suggestion
After the pod is created, we can see that all the Pods are landed on that Nodes and are in the `Running` state:
```

> After the Pod is created, we can see that all the Pods are landed on that Node(s) and are in the `Running` state:

Reviewer: ```suggestion
After the Pod is created, we can see that all the Pods are landed on that Node(s) and are in the `Running` state.
```
